tests/integration/tf/test_ci_07_third_party_integration.py

```diff
@@ -0,0 +1,25 @@
+from testbook import testbook
+
+from tests.conftest import REPO_ROOT
+
+
+@testbook(
+    REPO_ROOT / "examples/07-Train-a-third-party-model-using-the-Merlin-Models-API.ipynb",
+    execute=False,
+)
+def test_func(tb):
+    tb.inject(
+        """
+        import os
+        os.environ["INPUT_DATA_DIR"] = "/raid/data/movielens"
+        """
+    )
```

Comment:
I dont think this works in the same way than the movielens notebooks.

In https://github.com/NVIDIA-Merlin/models/blob/main/examples/01-Getting-started.ipynb we have a variable
```
input_path = os.environ.get("INPUT_DATA_DIR", os.path.expanduser("~/merlin-models-data/movielens/"))
train, valid = get_movielens(variant="ml-1m", path=input_path)
```

The CI machine has following data available
```
    |-- movielens-1m
    |   |-- README
    |   |-- movies.dat
    |   |-- ratings.dat
    |   `-- users.dat
    |-- movielens-25m
    |   |-- README.txt
    |   |-- genome-scores.csv
    |   |-- genome-tags.csv
    |   |-- links.csv
    |   |-- movies.csv
    |   |-- ratings.csv
    |   `-- tags.csv
```

This notebooks doesnt use INPUT_DATA_DIR
```
train, valid = get_movielens(variant='ml-100k')
```

My suggestions is:
- let's remove the integration tests for now, that we can merge the example. I think it is fine, if the unittest covers it.
- Create a ticket to add integration tests, that we do not forget it
- Talk to @jperez999 to add movielens 100k to the integration test machine.

The goal of integration tests are that it doesnt require to download the dataset everytime and runs the CI with realdata. So we should uplod the data into the correct directory and update the example with correct paths

Reply:
Makes a lot of sense! 🙂 Will follow up with @jperez999, he is on holiday atm so asked on the ci channel, maybe someone will be able to help.

Removed the test for the time being and created an issue here

Task created #639